### ISSUE TITLE
White Grapes

### ISSUE THREAD

**Issue by Sunconure11** [GrowthcraftCE/Growthcraft-DesignDocs#2]
Extension of existing module, was posted here:

http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/wip-mods/2505072-growthcraft-community-edition-proper-1-7-10?comment=276

@Killerjdog51 @marcoasfonseca

Note: Red Grape refers to the existing grapes already.

**Comment by Sunconure11** [comment]
Direct document link:

https://docs.google.com/document/d/1X-5XQpBPHNnGEpocH7mczhoadChpvG4Rhf8U7Qgqq3M/edit

**Comment by Sunconure11** [comment]
https://www.dropbox.com/sh/pk00tzxq8m0gjts/AAC81ULN1esqcaNm4uVj_Agoa?dl=0

**Comment by IceDragon200** [comment]
Looks a bit on the large size, I guess we'll need to scale it down ingame?

Could you get the bottle cork looking something like this (texture wise?)
[![Champagne Cork](http://www.advancecork.com/gifs/champagne-cork.jpg)](http://www.advancecork.com/gifs/champagne-cork.jpg)

**Comment by Sunconure11** [comment]
Note: If this idea of @Killerjdog51's gets accepted, as it could count towards "duplicate crops" for the most part, there should be a config option to completely turn this off along with all associated products. Lets try and please everyone. Even the internet's butt.

**Comment by hron84** [comment]
@Sunconure11 I see some new grapes in 2.5.2, is it just some preparation for adding them, or they are obtainable somehow? I can get only plain old wine grapes from vines.

**Comment by Sunconure11** [comment]
Even I am not sure of that. Could be possible preparation, but I am not sure.

**Comment by killerjdog51** [comment]
Red Grapes and White Grapes were added in RC (release candidate) 8 i think or maybe RC 9. I think they were preparing, but I don't know for certain. IceDragon just said that he was nitpicking some possible features from version 2.6.x.

**Comment by killerjdog51** [comment]
Also, does anyone know if there is a way to compact a pixelized image? like I want to get a smaller version of wood planks, or something similar, to simulate the top of the cork. The problem is that I think I can get that kind of detail in a 2x2 space. I know that Crayfish's Model creator can do that, except thats for .json files.

**Comment by Sunconure11** [comment]
Do what I'm doing for wood and stuff for wheat. Copy and paste a small selection of the desired texture, and paste it on the model's texture.

Alternatively, color the desired part of the texture with wood's color scheme.

Plus on a model that small, you probably won't need a whole lot of detail, unless you want to blow up the texture map.

**Comment by killerjdog51** [comment]
yea, thats what I did to start with. the problem is that I can't put enough detail into a 2x2 area to resemble to top of a cork.

**Comment by killerjdog51** [comment]
I made the cork in the texture file look closer to the cork on the culture jar, to make the textures more consistant. https://www.dropbox.com/sh/pk00tzxq8m0gjts/AAC81ULN1esqcaNm4uVj_Agoa?dl=0

**Comment by Sunconure11** [comment]
That can work.

However, what software is that?

**Comment by killerjdog51** [comment]
for what? I made the textures in photoshop and the models/java files in techne. For the original screenshots of consept though I used Crayfish's model creator for 1.8. for the .java files however I used techne.

**Comment by Sunconure11** [comment]
Oh.

**Comment by killerjdog51** [comment]
I was supposed to use techne, right?

**Comment by Sunconure11** [comment]
Yes, you can use that, but the model can be converted, just send over the raw .tcn file along with its textures.

**Comment by killerjdog51** [comment]
Uhhhh, .tcn files? I only have .java files.

**Comment by Sunconure11** [comment]
If you saved the model somewhere, you should have them.

If not, load up the model, and press "save as", and put them in a suitable spot.

**Comment by killerjdog51** [comment]
I don't think I ever saved the model. I just exported everything as a .java file.

**Comment by Sunconure11** [comment]
Next time, save the models somewhere. This will help, as if you lose the models on dropbox, you can just use one from a backup.

**Comment by killerjdog51** [comment]
I see, thanks for the advice.

**Comment by Sunconure11** [comment]
NP
